packages/contracts/src.ts/index.ts

```diff
@@ -129,8 +129,15 @@ const allowedTransactionKeys: { [ key: string ]: boolean } = {
     customData: true, nodeId: true,
 }
 
-async function populateTransaction(contract: Contract, fragment: FunctionFragment, args: Array<any>): Promise<PopulatedTransaction> {
-    // If an extra argument is given, it is overrides
+
+export function isAlias(address:string):boolean {
+    address = address.replace('0x', '');
+    // shard - 4 zeroes, realm - 8 zeroes, num - typically no zeroes
+    return address.split('').filter(e => e === '0').length < 12;
```

Comment:
Wouldn't it be more cleaner if we use:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith